Makefile

```diff
@@ -85,6 +85,9 @@ eslint-branch: node_modules/eslint node_modules/eslint-plugin-react node_modules
 i18n-lint:
 	@echo "$(JS_FILES)" | sed 's/\([^ ]*\/test\/[^ ]* *\)//g' | xargs -n1 $(I18NLINT)
 
+mixedindentlint: node_modules/mixedindentlint
+	@echo "$(JS_FILES)\n$(SASS_FILES)" | xargs mixedindentlint
```

Comment:
Also, there's 67 instances of mixed indentation in JS and JSX files currently so the eslint linter clearly isn't finding everything.